Tests/SpiderTest.php

```diff
@@ -4,12 +4,16 @@
 
 use PHPUnit_Framework_TestCase;
 use Simgroep\ConcurrentSpiderBundle\Spider;
+use Simgroep\ConcurrentSpiderBundle\CrawlJob;
 use VDB\Uri\Uri;
 use Symfony\Component\EventDispatcher\GenericEvent;
 
 class SpiderTest extends PHPUnit_Framework_TestCase
 {
-    public function testIfDoubleSlashIsRemoved()
+    /**
+     * @test
+     */
+    public function isDoubleSlashIsRemoved()
```

Comment:
change name of method to begin with test that everything is consistent, and things without test on beginning is internal used for phpunittests